### ISSUE TITLE
Report code coverage for modules tested with the Roxy Unit Tests

### ISSUE THREAD

**Issue by hansenmc** [marklogic/roxy#759]
It would be helpful to report and visualize code coverage for the modules that are tested by Roxy Unit Tests. 

This would help highlight areas that have not (yet) been covered and need additional tests, and could be used as project quality metrics.

It might be possible to incorporate, or at least model an implementation from the code coverage implemented in [xray 2.1 branch](https://github.com/robwhitby/xray/tree/v2.1) by @mblakele.

**Comment by grtjn** [comment]
@hansenmc Did you manage to get the code coverage in xray 2.1 running with some of the Roxy code? It is not well documented, and not fully complete from the looks of it..

**Comment by hansenmc** [comment]
No, I haven't worked on it yet. But I think it would be really useful! 

I'll update if I make any progress or will submit a PR if I can get anything working.

**Comment by grtjn** [comment]
Any working example of xray code coverage would be helpful.. :)
